ui/src/time_picker.tsx

```diff
@@ -228,19 +256,24 @@ export const
                   <Fluent.FocusTrapZone isClickableOutsideFocusTrap>
                     <Toolbar
                       setOpenView={setOpenView}
-                      time={parsedValue ? formatDateToTimeString(parsedValue as D, ampm ? '12' : '24') : parsedValue as null}
+                      time={
+                        parsedValue
+                          ? formatDateToTimeString(value && isOutOfBounds(parsedValue as D, minTime, maxTime) ? value : parsedValue as D, ampm ? '12' : '24')
+                          : null
+                      }
                       label={label}
                       switchAmPm={switchAmPm}
+                      amPmDisabled={value ? !canSwitchAmPm(value, minTime, maxTime) : false}
```

Comment:
No need since `amPmDisabled` prop is no more present after removing custom AM/PM switch controls.
Btw: Is it the same logic? If `value` resolves to `false`, my solution returns `false` but yours returns `true`.

Reply:
> Is it the same logic? If value resolves to false, my solution returns false but yours returns true.

You are right, copilot failed me. The point is ternaries that return booleans can be simplified in general as they are easier to figure out.

Reply:
Yep, I get the point. 

I see this was the mistake of a copilot, I've tried it with a ChatGPT and it does a same mistake:
![image](https://github.com/h2oai/wave/assets/23740173/ab2950f8-b6c3-4d9f-8e5f-687f68ee3df0)